internal/v1/handler_compose_image.go

```diff
@@ -1256,5 +1277,53 @@ func (h *Handlers) buildCustomizations(ctx echo.Context, cr *ComposeRequest, d *
 		}
 	}
 
+	if cust.Aap != nil {
+		// TODO: this doesn't handle the TLS certificate case yet.
+		// Also, the job template id might need to go into the API url
+		data := common.ToPtr(fmt.Sprintf(
+			"#!/usr/bin/bash\ncurl -s -i --data %q %s",
```

Comment:
This is based on:
https://github.com/datacenter/cloudcenter-content/blob/bbb41be37f2b306d5fbf28f8a5c8b2a679104752/services/ansible/request_tower_configuration.sh#L3-L8

Reply:
I would probably use `text/template` and `embed` Go packages here as the resulting code and templates are generally more readable and we plan to refactor Satellite registration this way as well so a utility template package could be a great fit. I do not insist tho.